fvm/context.go

```diff
@@ -87,13 +87,11 @@ func defaultContext(logger zerolog.Logger) Context {
 		CadenceLoggingEnabled:         false,
 		EventCollectionEnabled:        true,
 		ServiceEventCollectionEnabled: false,
-		AccountFreezeAvailable:        false,
+		AccountFreezeEnabled:          true,
```

Comment:
Why did the default change?

Reply:
I actually changed the semantic of this parameter, 
now we don't change this based on if an account is payer or not, we can use it as a high level flag to enable and disable the feature.